py/selenium/webdriver/common/action_chains.py

```diff
@@ -61,7 +70,7 @@ class ActionChains:
     another.
     """
 
-    def __init__(self, driver, duration=250, devices=None):
+    def __init__(self, driver, duration: int = 250, devices: Optional[List[any_device]] = None):
```

Comment:
And if you use typing extensions , you may be able to do `List[any_device] | None`